### ISSUE TITLE
Unable to verify rufus executable with signature file

### ISSUE THREAD

**Issue by StKob** [pbatard/rufus#1287]
To verify file I tried using kleopatra and gpg from wsl. I wasn't able to verify it. 

Thats what I'm getting from gpg. 
```
gpg --verify rufus-3.4.exe.sig rufus-3.4.exe
gpg: packet(3) with unknown version 137
gpg: no signature found
gpg: the signature could not be verified.
Please remember that the signature file (.sig or .asc)
should be the first file given on the command line.
```

Kleopatra was telling "Unsuccessful check: No data"

**Comment by pbatard** [comment]
The `.sig` is __NOT__ a GPG signature or anything that can be validated using external tools unless you know exactly what you are doing. Instead it is a __custom__ RSA signature that is only meant to be used __internally__ by Rufus.

If you really want to know how it's used you can have a look [here](https://github.com/pbatard/rufus/blob/master/src/net.c#L509-L573) and [here](https://github.com/pbatard/rufus/blob/master/src/pki.c#L654-L733).

Also the executable is digitally signed, so there is exactly zero need to manually validate the signature as this is being carried out automatically by Windows before it executes the file. Unlike what might be the case for non executable files, executable files can be digitally signed using standard mechanisms, and Rufus also makes use of those. Please read on Authenticode if you want to know more and understand why you don't need to bother with asking the author of a digitally signed Windows executable for an SHA or a digital signature.

In other words, the `.sig` is just an __internal__ file that has nothing to do with GPG `.sig` files, and that users do not __ever__ need to concern themselves about. As a matter of fact, we used to not list these files from our servers, but since our releases are now hosted by GitHub we can no longer do that.
